CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 ## 0.10.5
   - Features Added
-    - Recommended node.js version to build Alchemy is set to 12.16.1.
+    - change home page to show DAOs again instead of feed. Show separate "Your DAOs" section containing DAOs the user follows or is a member of, then the list of other DAOs below.
+    - added support for embedded youtube and vimeo videos in proposal descriptions
+    - recommended node.js version to build Alchemy is set to 12.16.2
+    - upgrade to use subgraph v39_3
```

Comment:
#1619 need to be merged to support subgraph v39_3. please merge it

Reply:
done